test/bats.bats

```diff
@@ -36,6 +36,12 @@ fixtures bats
   [ $(expr "$output" : ".*does not exist") -ne 0 ]
 }
 
+@test "missing script for --global-helper prints error and usage instructions" {
+  run bats --global-helper=
+  [ $status -eq 1 ]
+  [ $(expr "$output" : ".*You must supply a script with --global-helper.*") -ne 0 ]
```

Comment:
I think you can get away with an actual equality statement here, avoiding the need for `expr`.